src/commands/push.js

```diff
@@ -154,6 +156,14 @@ class PushCommand extends ApifyCommand {
 
         outputs.link('Actor build detail', `https://console.apify.com/actors/${build.actId}#/builds/${build.buildNumber}`);
 
+        const shouldOpenBrowser = await inquirer.prompt([
+            { type: 'confirm', name: 'continue', message: 'Do you want to open build detail in your browser?', default: true },
+        ]);
+
+        if (shouldOpenBrowser.continue) {
+            open(`https://console.apify.com/actors/${build.actId}`);
```

Comment:
This will not work when the user is logged in as an organization, you'll have to add the `/organization/<USER-ID>` to the URL in that case.

Reply:
I see that this is the case also for the link at line 157, so we should adjust it there.